public/pages/Alerts/containers/Alerts/Alerts.tsx

```diff
@@ -199,7 +207,39 @@ export class Alerts extends Component<AlertsProps, AlertsState> {
         name: 'Status',
         sortable: true,
         dataType: 'string',
-        render: (status) => (status ? capitalizeFirstLetter(status) : DEFAULT_EMPTY_DATA),
+        render: (status, alertItem) => {
+          if (status === 'ERROR') {
+            const isModalVisible = this.state.isModalVisible;
+            return (
+              <React.Fragment>
+                <EuiLink onClick={() => this.setState({ isModalVisible: true })}>
+                  {capitalizeFirstLetter(status)}
+                </EuiLink>
+
+                {isModalVisible && (
+                  <EuiModal onClose={() => this.setState({ isModalVisible: false })}>
+                    <EuiModalHeader>
+                      <EuiModalHeaderTitle>{capitalizeFirstLetter(status)}</EuiModalHeaderTitle>
+                    </EuiModalHeader>
+
+                    <EuiModalBody>
+                      <EuiIcon type="alert" size="l" />
+                      <p>{alertItem.error_message}</p>
```

Comment:
What happens if the `error_message` is null?

Reply:
> What happens if the `error_message` is null?

when the status is `ERROR`, the `error_message` ideally shouldn't be null but I will check this in the backend, or we can add another null check here.

Reply:
will be an empty string now